### PR TITLE
button and link tweaks – closes #273

### LINKED ISSUE #273: QA: Global Issue - Button and Links
- [x] **Yellow CTA Buttons** **IMPORTANT**
- Type: remains black on hover (not blue)
- Hover: subtle background #FFED00 changes to #FFE800 (slightly darker yellow)

- [x]  **Blue Border (Ghost) Buttons** 
- Type: 16px, bold nunito sans
- Border: 2px
- Hover: black text changes to white (on blue bg)

blue-ghost-button-hover

- [ ]  **Blue Narrow Buttons (more details)**
- Type: 12px, nunito sans
- Button Height: 24px 
- Hover: blue bg changes to darker blue bg #0081ad (darker blue)

image

- [x]  **Blue Link with Arrow**
- underline doesn’t go under the arrow
- Type: bold nunito sans
- Hover: #0081ad (darker blue)

link-hover

- [ ]  **Links**
- Type: bold nunito sans 
- Hover: #0081ad (darker blue)

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Note: I didn't touch the blue narrow buttons on the People page because I think you already did that...

File: source/sass/buttons.scss [hunk 1]
```diff
@@ -11,17 +11,22 @@
...
   }
 }
 
 .btn-normal {
   color: $black;
   background: #ffed00;
+
+  &:hover {
+    color: $black;
+    background: #ffe800;
```
Comment (alanmoo) [inline source/sass/buttons.scss]: This is ridiculously subtle

**Comment by alanmoo** [comment]
I'm not sure if this actually closes the issue, as the btn-sm doesn't have the hover color on the people page. We should probably set our button colors variables at the start of projects so that we can use the built in bootstrap classes instead of having to rebuild them every time.

File: source/sass/buttons.scss (hunk 1 above)
Comment (gvn) [inline source/sass/buttons.scss]: I agree `¯\_(ツ)_/¯`

**Pull request merged**